api/serializers/song.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from api.models import Song
+
+
+class SongSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Song
+        fields = "__all__"
```

Comment:
This is the serializer, it's what describes to DRF how to read the data out of the database and how to read it from the web and store it back into the database. We'll need one of these for `song_entry` and `songbook` as well.

Reply:
Oh, and to be more specific, it WOULD describe how to read the data but instead it rips all of that information out of the model (which happens because we're inheriting from `serializers.ModelSerializers` (emphasis on Model).